java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -1709,6 +1710,31 @@ public void zcard_returns_success() {
         assertEquals(value, payload);
     }
 
+    @SneakyThrows
+    @Test
+    public void zscore_returns_success() {
+        // setup
+        String key = "testKey";
+        String member = "testMember";
+        String[] arguments = new String[] {key, member};
+        Double value = 3.5;
+
+        CompletableFuture<Double> testResponse = new CompletableFuture<>();
+        testResponse.complete(value);
```

Comment:
shouldnt it be?

```
CompletableFuture<Double> testResponse = mock(CompletableFuture.class);
when(testResponse.get()).thenReturn(value);
```

Reply:
We are moving away from using mocks on the Future.  The line 
```
CompletableFuture<Double> testResponse = mock(CompletableFuture.class);
```
causes a compilation warning.  Nothing serious, but there's really no reason to mock it if we can avoid it.